ADAL/src/cache/ios/ADKeychainTokenCache.m

```diff
@@ -768,9 +724,20 @@ - (void)testRemoveAll:(ADAuthenticationError * __autoreleasing *)error
         OSStatus status = SecItemDelete((CFDictionaryRef)query);
         [ADKeychainTokenCache checkStatus:status operation:@"remove all" correlationId:nil error:error];
         
-        // Remove the tombstone timestamp as well;
-        status = SecItemDelete((CFDictionaryRef)_defaultTombstone);
-        [ADKeychainTokenCache checkStatus:status operation:@"remove tombstone timestamp" correlationId:nil error:nil];
+        //JASON
+        query =
+        [@{
+           (id)kSecClass                : (id)kSecClassGenericPassword,
+           (id)kSecAttrGeneric          : [s_wipeLibraryString dataUsingEncoding:NSUTF8StringEncoding],
+           (id)kSecAttrAccessGroup      : _sharedGroup,
+           (id)kSecAttrAccount          : @"TokenWipe",
+           (id)kSecReturnData           : @YES
+           } mutableCopy];
```

Comment:
perhaps I'm missing something, does it need to be mutable?

Reply:
design decision, there are places where this will be changed,
when fetching the log, it needs to have an additional object for kSecReturnData. 
same for adding the log for the first time as well.

Reply:
Hmm, calling mutable copy when needed seems more logical the more I think about this.